prettier.json

```diff
@@ -0,0 +1,3 @@
+{
+  "proseWrap": false
+}
```

Comment:
Rename as `prettier.json` since `.prettierrc` will cause `--find-config-path` tests to fail.

Reply:
We should fix the tests (maybe mock impl for `existsSync()`?